### PR TITLE
Update telemetry for 5.20

### PR DESCRIPTION
**Pull request by jasonblais**

### CODE REVIEW

File: source/administration/telemetry.rst [hunk 1]
```diff
@@ -84,6 +84,7 @@ Non-personally Identifiable Diagnostic Information, distinguished by end users a
   - *Commercial License Diagnostics (Enterprise Edition Only):* Uploaded an Enterprise license key to the server
   - *Mobile Performance Diagnostics:* Load times for starting the app, switching channels, and switching teams 
   - *Permissions Discovery Diagnostics (Enterprise Edition Only):* Provides all the permissions configured for each role for the System Scheme and each Team Override Scheme created in the system.  Scheme ID; Team Admin Permissions; Team User Permissions; Channel Admin Permissions; Channel User Permissions; Number of teams the scheme is associated with
+  - *Group Discovery Diagnostics (Enterprise Edition Only):* Provides information related to AD/LDAP groups, including number of groups synced to Mattermost, ...
```
Comment (jasonblais) [inline source/administration/telemetry.rst]: @mkraft Can you help share what telemetry is collected for groups based on your PR here? https://github.com/mattermost/mattermost-server/pull/13736

Comment (mkraft) [reply source/administration/telemetry.rst]: * group_count (all groups)
* group_team_count (teams associated to groups)
* group_channel_count (channels associated to groups)
* group_synced_team_count (teams that are group-synced)
* group_synced_channel_count (channel that are group-synced)
* group_member_count (all groups, including duplicate user ids)
* distinct_group_member_count (all groups, excluding duplicate user ids)

Comment (jasonblais) [reply source/administration/telemetry.rst]: Thanks! Updated PR accordingly

**Comment by jasonblais** [comment]
Will go ahead and merge

**Pull request merged**